### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by Lovor01** [tomayac/opfs-explorer#19]
It worked for me initially, but then stopped working. I tested on [wordpress playground](https://playground.wordpress.net)

**Comment by tomayac** [comment]
Could you try the reset step [outlined in the FAQ](https://github.com/tomayac/opfs-explorer?tab=readme-ov-file#faq)?

**Comment by Lovor01** [comment]
Hey, thanks for quick answer. Yes I tried it. But the problem is of different nature - tab does not disappear, instead folder structure is not refreshed. I deleted everything inside root in OPFS in hope that this would reset things. But it didn't, I now have only root folder and nothing more. Funny, your example page works fine, but wordpress playground isn't. And I am quite sure it is using OPFS, otherwise it wouldn't work.
